master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Using etcd RBAC to segment Kubernetes and Calico
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes the users and roles
+needed to segment etcd so that Kubernetes and Calico can only read and write
+within their respected subtrees/prefixes.  To configure more compartmentalized
+configurations of the Calico components, see this addon
+[guide](kubernetes-advanced).
+
+This guide assumes you are following the general
+[Setting up etcd certificates for RBAC guide](index) and using its guidance
+for setting up certificates and etcd cluster, users, and roles.
+
+## Why you might be interested in this guide
+
+You are using Kubernetes and Calico who share an etcd datastore and wish
+to ensure that Calico and Kubernetes are unable to access each other's etcd
+data.
+
+## Needed etcd Roles
+
+The following components need certificates with a Common Name that matches an
+etcd user that has been given appropriate roles allowing access to the key
+prefixes or paths listed below.
+- kube-apiserver
+  - Read and Write access to `/registry`.
+    - With Kubernetes v1.6 the etcd v3 API is used, this can be switched
+	  to etcd v2 by passing a flag to kube-apiserver.
+	- This key prefix can be changed by passing the proper option to the
+	  kube-apiserver.
+  - The etcd user needs to be given the root role to perform compaction.
+- Calico
+  - Read and Write access to `/calico`.
+  - Calico uses the etcd v2 API so the user and roles must be created there
+    too.
+
+All certificate/key pairs that are referenced below are assumed to have been
+created for the specific component with the information above.
+
+## Kubernetes API server
+
+The kube-apiserver is the only Kubernetes component that directly accesses etcd.
+The flags required to provide the kube-apiserver with certificates for
+accessing an etcd cluster are:
+- `--etcd-cafile=<CA certificate`
+- `--etcd-certfile=<certificate with etcd username as CN>`
+- `--etcd-keyfile=<key for the above certificate>`
+Setting these will depend on the method used to deploy Kubernetes so refer
+to your integrator's documentation for help setting these flags.
+
+## Updating a hosted Calico manifest
+
+With the Calico certificate and key,
+[this manifest]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/hosted/calico.yaml)
+can be updated with the CA, certificate, and key information and then used to
+deploy hosted Calico.  The same information could be added to or updated in
```

Comment:
First sentence repeats itself a little.  How about:

"To deploy hosted Calico with the CA, and Calico-specific certificate and key, use this *manifest template* with the modifications described below".